react/src/pages/analysis/Analysis.tsx

```diff
@@ -107,9 +108,10 @@ const analyses = [
 
 export default function Analysis() {
     const classes = useStyles();
-    const [detail, setDetail] = useState(false);
+    let { analysisID } : any = useParams();
```

Comment:
If you use the angle bracket syntax to specify the type like below, `any` can be avoided. Also I think you can use `const` here.